### PR TITLE
Add swagger for all apis

### PR DESCRIPTION
**Pull request by 0123meir**

### CODE REVIEW

File: src/Controllers/users.js [hunk 1]
```diff
@@ -9,6 +9,13 @@ const {
   updateUserById,
 } = require("../DAL/users");
 
+const extractUserProps = (user) => ({
```
Comment (AsafL910) [inline src/Controllers/users.js:12]: why is this needed?

**Review by AsafL910 — CHANGES_REQUESTED** [review]

File: src/Controllers/users.js (hunk 1 above)
Comment (0123meir) [reply src/Controllers/users.js:12]: I don't want to return passwords as part of the user object, maybe there is a better way to extract props in-case the user object changes, but I think it's fine for the scope of this assignment
